skimage/feature/texture.py

```diff
@@ -31,10 +31,10 @@ def greycomatrix(image, distances, angles, levels=None, symmetric=False,
         The input image should contain integers in [0, `levels`-1],
         where levels indicate the number of grey-levels counted
         (typically 256 for an 8-bit image). This argument is required for
-        16-bit images or higher and is typically the maximum of the image. 
+        16-bit images or higher and is typically the maximum of the image.
         As the output matrix is at least `levels` x `levels`, it might
-        be preferable to use binning of the input image rather than 
-        large values for `levels`. 
+        be preferable to use binning of the input image rather than
+        large values for `levels`.
```

Comment:
@jni Can we just add that if the image type is uint8, then levels is set to 256?